.travis.yml

```diff
@@ -2,13 +2,14 @@ os: osx
 osx_image: xcode10.1
 language: objective-c
 install: true
-addons:
-  homebrew:
-    taps: homebrew/cask-versions
-    casks: java8
 before_install:
-    - brew update
-    - brew install bazel
+    - bazel_version=0.22.0
```

Comment:
Wonder if we should delay this PR until this version is available, so that we don't end up changing the version here every new release?

Reply:
The current Bazel version via Homebrew is 0.20.0, which is 2 version behind the latest version (0.22.0) in Bazel's GitHub repo. Additionally, being dependent on Homebrew may prevent us from updating to the latest versions of Apple and Swift rules. This actually happens to be the case currently where Apple rules v0.13.0 does not work with Bazel v0.20.0.

We don't necessarily have to update this version each release. Can update Apple rules and xctestrunner versions more frequently as long as they work with the Bazel version specified here. Manually downloading a given version of the binary installer provides us with the flexibility to update to the latest version, if needed, without having to rely on Homebrew.